nilearn/glm/first_level/first_level.py

```diff
@@ -824,6 +825,11 @@ def first_level_from_bids(dataset_path, task_label, space_label=None,
         Specifies the space label of the preprocessed bold.nii images.
         As they are specified in the file names like _space-<space_label>_.
 
+    sub_labels : list of str, optional
+        Specifies the subset of subject labels to model.
+        If 'None', will model all subjects in the dataset.
+        .. versionadded:: 0.9.3.dev
```

Comment:
I may be wrong but this version added should have been updated, no?

Reply:
Oops missed that. Yea it should be 0.10.1.dev

Reply:
will fix in #3525